### PR TITLE
chore: Wrap long lines in plugins/parsers

### PR DESCRIPTION
**Pull request by zak-pawel**
Wrap lines longer than 160 characters.

It is only part of the bigger job.
After long lines in whole project are handled, we can enable proper linter to guard this.

### CODE REVIEW

File: plugins/parsers/grok/influx_patterns.go [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package grok
 
+//nolint:lll,revive // conditionally long lines allowed
```
Comment (powersj) [inline plugins/parsers/grok/influx_patterns.go:3]: why is this needed? the example_log line is less than 160?

Comment (powersj) [reply plugins/parsers/grok/influx_patterns.go:3]: ok thanks! looks like I didn't look for enough down to find the long line

**Review by powersj — APPROVED** [review]

**Review by MyaLongmire — APPROVED** [review]
